shared/utils/yaml-to-shorthand.py

```diff
@@ -10,13 +10,29 @@
 import codecs
 import sys
 
+
 try:
     from xml.etree import cElementTree as ET
 except ImportError:
     import cElementTree as ET
 
 
+def construct_yaml_bool(self, node):
+    value = self.construct_scalar(node)
+
+    return value
+
+
+def bool_constructor(self, node):
+    value = construct_yaml_bool(self, node)
+
+    return value
```

Comment:
this is python code, not a Tolkien novel :-P Please do this on just 2 lines.

Reply:
But @mpreisler I like Tolkien novels... maybe that is why it is written as such. :D

Reply:
@mpreisler Tolkien novel reduced to Ernest Hemingway's shortest novel.